src/views/Farms/components/FarmCard/ApyButton.tsx

```diff
@@ -72,21 +80,25 @@ const ApyButton: React.FC<React.PropsWithChildren<ApyButtonProps>> = ({
       <Text>
         {t('APR (incl. LP rewards)')}:{' '}
         <Text style={{ display: 'inline-block' }} color={strikethrough && 'secondary'}>
-          {strikethrough ? `${(apr * 2 + lpRewardsApr).toFixed(2)}%` : `${displayApr}%`}
+          {strikethrough ? `${(apr * boostMultiplier + lpRewardsApr).toFixed(2)}%` : `${displayApr}%`}
```

Comment:
Hmm. It's good to show the specific boostMultiplier. However, if the proxy address hasn't been created - user hasn't enabled the proxy farm, I think we don't have a specific boostMultiplier value to show. That's why we show the maximum instead.

Reply:
When the proxy address hasn't been created, or user hasn't enabled the proxy farm, it's still show 2x boostMultiplier, so it won't change.

Reply:
Got it. Thanks for your contribution.